docs/DeveloperGuide.md

```diff
@@ -244,6 +244,40 @@ The following activity diagram summarizes what happens when a user executes a ne
     * Pros: Will use less memory (e.g. for `delete`, just save the person being deleted).
     * Cons: We must ensure that the implementation of each individual command are correct.
 
+## `AddAnimalCommand` Implementation
+
+### Proposed Implementation
+
+The `AddAnimalCommand` is a specific command designed to add an animal to the animal catalog. It identifies the animal to be added with attributes such as its name, ID, species, breed, age, date of birth and date of admission.
+
+Here's a brief outline of its operations and attributes:
+
+- `AddAnimalCommandParser#parse#` — Parses the user input to create an animal with the given attributes.
+- `AddAnimalCommand#execute(AnimalModel model)` — Executes the command to add a specified animal to the model.
+
+Given below is an example usage scenario of the `AddAnimalCommand`:
+
+1. The user types in the `add` command with the proper cli syntax, giving a compulsory input field for each attribute. Note that the position of the cli syntax can be in any order.
+2. The command verifies the validity of the index. If the index is missing any of the required syntax, it throws a `CommandException`. Otherwise, it adds the animal with its inputted attributes to the model and returns a successful command result.
+
+The following sequence diagram shows how the `AddAnimalCommand` works:
+
+<puml src="diagrams/DeleteSequenceDiagram.puml" alt="AddSequenceDiagram" />
```

Comment:
Maybe you forgot to switch DeleteSequenceDiagram to Add !

Reply:
You are right! Fixed!